### PR TITLE
Handle invalid price level updates gracefully.

### PR DESCRIPTION
**Pull request by vincentwschau**
### Changelist
Don't throw errors when invalid price level updates occur, instead set price level to 0.

### Test Plan
N/A

### Author/Reviewer Checklist
- [ ] If this PR has changes that result in a different app state given the same prior state and transaction list, manually add the `state-breaking` label.
- [ ] If the PR has breaking postgres changes to the indexer add the `indexer-postgres-breaking` label.
- [ ] If this PR isn't state-breaking but has changes that modify behavior in `PrepareProposal` or `ProcessProposal`, manually add the label `proposal-breaking`.
- [ ] If this PR is one of many that implement a specific feature, manually label them all `feature:[feature-name]`.
- [ ] If you wish to for mergify-bot to automatically create a PR to backport your change to a release branch, manually add the label `backport/[branch-name]`.
- [ ] Manually add any of the following labels: `refactor`, `chore`, `bug`.

### CODE REVIEW

**Review by BrendanChou — APPROVED** [review]

File: indexer/packages/redis/__tests__/caches/orderbook-levels-cache.test.ts [hunk 1]
```diff
@@ -192,30 +191,24 @@ describe('orderbookLevelsCache', () => {
 
       // Test that an error is thrown if the update results in a negative quantums for the price
```
Comment (Christopher-Li) [inline indexer/packages/redis/__tests__/caches/orderbook-levels-cache.test.ts:192]: nit: remove

**Comment by vincentwschau** [comment]
@Mergifyio backport indexer/release/v5.0.4

**Comment by vincentwschau** [comment]
@Mergifyio backport indexer/release/v5.0.4

**Comment by vincentwschau** [comment]
@Mergifyio backport release/indexer/v5.0.4

**Pull request merged**